### ISSUE TITLE
converting voc->coco for segmentation task 

### ISSUE THREAD

**Issue by jihyeonyi** [openvinotoolkit/datumaro#229]
I've converted the voc data to coco format for segmentation task and tried to train OTE instance segmentation models with the data.
```
voc.export(out_path, "coco", save_images=True, reindex=True)
```

However, OTE training didn't proceed anymore after outputting this log.
```
2021-04-28 17:09:50,822 - mmdet - INFO - workflow: [('train', 1)], max: 12 epochs
```
OTE can process the original coco data, which consists of mostly polygons.
In contrast, the converted data consists of RLE masks. 
So, I've converted VOC masks to polygons before exporting them to coco format.
```
converted = Dataset(MasksToPolygons(voc))
converted .export(out_path, "coco", save_images=True, reindex=True)
```
Finally, OTE training can be done.

I'm not sure that OTE has a problem with handling RLE masks, or datumaro can't generate proper RLE masks.
So I report this.

**Comment by jihyeonyi** [comment]
For your information, the OTE model I've tried is `efficientnet_b2b-mask_rcnn-480x480` at https://github.com/openvinotoolkit/training_extensions/tree/develop/models/instance_segmentation/model_templates/custom-instance-segmentation

**Comment by zhiltsov-max** [comment]
I don't see any mentions of RLE in https://github.com/openvinotoolkit/mmdetection/blob/ae96c630ab9ec66d6c7fd0fb1bd2ed5eaf9c928f/mmdet/datasets/coco.py, which should be used for this format, so I assume it is not supported. 

@druzhkov-paul, can you confirm?

**Comment by zhiltsov-max** [comment]
@druzhkov-paul, does OTE support RLE masks?

**Comment by druzhkov-paul** [comment]
AFAIK, it has a limited support of RLE masks, which may be incomplete when masks as they are represented in annotation are not converted to binary masks at a pre-processing stage. This is exactly the case with model's config referenced above: [see here](https://github.com/openvinotoolkit/training_extensions/blob/develop/models/instance_segmentation/model_templates/custom-instance-segmentation/efficientnet_b2b-mask_rcnn-480x480/model.py#L128).

**Comment by jihyeonyi** [comment]
Thank you @druzhkov-paul 
As I understand, it could be solved if I use or implement the correct annotation loader.
Is it correct?

**Comment by druzhkov-paul** [comment]
Correct. I'd say, the easiest thing to try is to set `poly2mask=True` in the config file. If this does not resolve the issue, then changes in the code of pre-processing pipeline are required.

Maybe @DmitriySidnev, who's developed the aforementioned model is more knowledgeable about possible solutions or workarounds.
